### ISSUE TITLE
Use of definition list for tags

### ISSUE THREAD

**Issue by imperator99** [govau/uikit#367]
- [X] I’ve read and understood the [Contributing guidelines](./CONTRIBUTING.md) and have done my best effort to follow them.
- [X] I’ve read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

# What happened

Accessibility testing of beta.dta.gov.au identifies the use of `<dl>` and `<dd>` to create tag lists (as per [the Design System tags component](https://designsystem.gov.au/components/tags/) as a possible accessibility issue. This was identified through expert observation so it doesn't actually break any accessibility guidelines _per se_. The accessibility tester does not believe a list of tags constitutes a definition list.

# What I expected to happen

I suggest further research be done on the appropriateness of using the `<dl>`, `<dt>` and `<dd>` tags for the tags component as opposed to the `<ul>` and `<li>` tags. If the use of a definition list is maintained, the reasoning behind this should be added to the rationale.

**Comment by TrebBrennan** [comment]
Thanks @imperator99, we've passed this on to our internal accessibility experts for feedback as well.

In the meantime the tags component already work with `<ul>` if you'd prefer to move to that.

You can see an example of that under [tags without links](https://designsystem.gov.au/components/tags/#tags-without-links) or checkout 
[this test page](https://uikit.service.gov.au/packages/tags/tests/site/)

**Comment by TrebBrennan** [comment]
Internal accessibility experts confirm.
**Recommend: Tags using definition to be deprecated.**

**Comment by pattyde** [comment]
TODO:

Remove all references to `<dl>` and `<dd>` tags in the tags component
